packages/validators/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@contentful/experiences-validators",
+  "private": true,
+  "version": "0.0.0",
+  "main": "./dist/index.js",
+  "module": "./dist/index.js",
+  "types": "./dist/types.d.ts",
+  "type": "module",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/contentful/experience-builder.git"
+  },
+  "scripts": {
+    "dev": "vite",
+    "clean": "rimraf dist",
+    "prebuild": "npm run clean",
+    "build": "rollup -c rollup.config.mjs",
+    "preview": "vite preview",
+    "test": "vitest",
+    "test:coverage": "vitest run --coverage"
+  },
+  "files": [
+    "dist/**/*.*"
+  ],
+  "typesVersions": {
```

Comment:
I'm just reading that this kinda the old way of defining types exports and than one should use `exports` instead/ additionally (see [here](https://stackoverflow.com/a/75738953/4816930)). Looking at the `core` module, I think this would a valid config:
```suggestion
  "exports": {
    ".": {
      "types": "./dist/index.d.ts",
      "import": "./dist/index.js"
    }
  },
  "typesVersions": {
```
We might not need to explicitly define the `types` property as eventually, it will point automatically to the `index.d.ts` based on the exported `index.js`. We can still go this way though to be safe.

Reply:
For reference, here you see that we have the `index.d.ts` which has the exact same content as the `types.d.ts`
![Screenshot 2024-02-12 at 16 06 25](https://github.com/contentful/experience-builder/assets/9327071/5073400e-322b-48bb-af6e-221899170a41)